File: objc/Builder/Builder/Computer.m
#import "Computer.h"

@implementation Computer

-(void)printSpec {
    NSLog(@"------------------------------------------------");
    NSLog(@"CPU: %@", _cpu);
    NSLog(@"Motherboard Model: %@", _motherboardModel);
    NSLog(@"Ram Value: %@", _ramValue);
    NSLog(@"Storage Value: %@", _storageValue);
    NSLog(@"Graphics Card Model: %@", _graphicsCardModel);
    NSLog(@"------------------------------------------------");
}

@end

File: swift/Builder/Builder/Computer.swift
import Foundation

public class Computer {
    var cpu: String?
    var motherboardModel: String?
    var ramValue: String?
    var storageValue: String?
    var graphicsCardModel: String?
    
    func printSpec() -> Void {
        print("------------------------------------------------");
        print("CPU: \(cpu!)");
        print("Motherboard Model: \(motherboardModel!)");
        print("Ram Value: \(ramValue!)");
        print("Storage Value: \(storageValue!)");
        print("Graphics Card Model: \(graphicsCardModel!)");
        print("------------------------------------------------");
    }
}
